sources/packages/web/src/services/http/StudentAppealApi.ts

```diff
@@ -45,4 +49,19 @@ export class StudentAppealApi extends HttpBaseClient {
       this.handleAPICustomError(error);
     }
   }
+
+  /**
+   * Gets all pending student application exceptions.
```

Comment:
The comment says `exceptions` on a `getPendingAppeals`, is it right?

Reply:
Error, I will update.